frontend/assets/javascripts/src/modules/ccpa.js

```diff
@@ -0,0 +1,37 @@
+// ----- Imports ----- //
+
+import { Raven } from 'src/modules/raven';
+
+// ----- Functions ----- //
+
+const GEOCOUNTRY_URL = '/geocountry';
+const US_COUNTRY_CODE = 'US';
+
+let fetchCountry;
+
+export const ccpaEnabled = () => {
+    const useCCPA = true; // set false to switch CCPA off
+
+    if (!useCCPA) {
+        return Promise.resolve(false);
+    }
+
+    if (fetchCountry) {
+        return fetchCountry;
+    }
+
+    fetchCountry = fetch(GEOCOUNTRY_URL).then(response => {
+        if (response.ok) {
```

Comment:
we probably need to make the URL return an InternalServerError rather than Ok if the header was not set.  Then we will get an alarm if this breaks.
We probably don't want to stop showing it altogether if the header disappears.